cob_cartesian_controller/include/cob_cartesian_controller/cartesian_controller_data_types.h

```diff
@@ -43,7 +43,6 @@ namespace cob_cartesian_controller
 
     struct MoveLinStruct
```

Comment:
I was thinking about that, but I think I'm ok with having struct member without trailing "_" in order to really highlight class members....